shell-completion/Makefile

```diff
@@ -1,29 +1,31 @@
+MAKEFLAGS += --jobs=$(shell nproc)
 
 .PHONY: command-options clean
 
 all: hledger-completion.bash
 
-hledger-completion.bash: hledger-completion.bash.m4 commands-list.txt query-filters.txt generic-options.txt command-options
```

Comment:
`commands-list.txt` is included by m4, so it should be in the prerequisites list too

Reply:
Roger that.